.github/workflows/R-CMD-check.yaml

```
@@ -31,22 +31,27 @@ jobs:
       R_KEEP_PKG_SOURCE: yes
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Harden the runner (Audit all outbound calls)
+        uses: step-security/harden-runner@4d991eb9b905ef189e4c376166672c3f2f230481 # v2.11.0
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)